src/redux/hooks/blocksHooks.ts

```diff
@@ -0,0 +1,21 @@
+import { useAppSelector } from '@redux/hooks/appHooks';
+import { BLOCK_NAMESPACE, IBlockState } from '@redux/reducers/blockReducer';
+
+// eslint-disable-next-line
+export function useBlockSelector<T extends (_state: IBlockState) => any>(
```

Comment:
Really bad to have `any` in this place, so components works with unpredictable data
Please fix

Reply:
I just fixed it.
This commit
https://github.com/pastelnetwork/pastel-explorer-frontend/pull/110/commits/85258d89013d4fa0294078643721f092e9528ebf